streams.json

```diff
@@ -5513,135 +7372,155 @@
           "contacts": [
               "ebernard@redhat.com",
-              "gbadner@redhat.com"
+            "gbadner@redhat.com"
           ],
           "repository_type": "GIT",
-          "repository_url": "https://github.com/hibernate/hibernate-search",
+          "repository_url": "https://github.com/hibernate/hibernate-search.git",
           "codebase": "5.5",
           "tag": "5.5.8.Final",
+          "group_id": "org.hibernate",
+          "artifact_id": "hibernate-search-parent",
           "version": "5.5.8.Final-redhat-1",
-          "gav": "org.hibernate",
+          "gav": "hibernate-search-parent:org.hibernate:5.5.8.Final-redhat-1",
           "comment": ""
         },
         {
           "component_name": "hibernate-hibernate-validator",
           "contacts": [
               "ebernard@redhat.com",
-              "gbadner@redhat.com"
+            "gbadner@redhat.com"
           ],
           "repository_type": "GIT",
-          "repository_url": "https://github.com/hibernate/hibernate-validator",
+          "repository_url": "https://github.com/hibernate/hibernate-validator.git",
           "codebase": "5.3",
           "tag": "5.3.5.Final",
+          "group_id": "org.hibernate",
+          "artifact_id": "hibernate-validator-parent",
           "version": "5.3.5.Final-redhat-2",
-          "gav": "org.hibernate",
+          "gav": "hibernate-validator-parent:org.hibernate:5.3.5.Final-redhat-2",
           "comment": ""
         },
         {
           "component_name": "hibernate-jpa-api",
           "contacts": [
               "ebernard@redhat.com"
           ],
-          "repository_type": "SVN",
-          "repository_url": "https://anonsvn.jboss.org/repos/hibernate/jpa-api/",
+          "repository_type": "",
+          "repository_url": "",
           "codebase": "trunk",
-          "tag": "tags/hibernate-jpa-2.0-api-1.0.1.Final",
-          "version": "1.0.1.Final",
-          "gav": "org.hibernate.javax.persistence",
+          "tag": "",
+          "group_id": "",
+          "artifact_id": "",
+          "version": "",
+          "gav": "",
           "comment": "NOTE: this is best guess."
         },
         {
           "component_name": "hornetq-hornetq",
           "contacts": [
               "jbertram@redhat.com",
-              "csuconic@redhat.com"
+            "csuconic@redhat.com"
           ],
           "repository_type": "GIT",
-          "repository_url": "https://github.com/hornetq/hornetq",
+          "repository_url": "https://github.com/hornetq/hornetq.git",
           "codebase": "2.4.x",
           "tag": "HornetQ_2_4_7_Final",
+          "group_id": "org.hornetq",
+          "artifact_id": "hornetq-pom",
           "version": "2.4.7.Final-redhat-1",
-          "gav": "org.hornetq",
+          "gav": "hornetq-pom:org.hornetq:2.4.7.Final-redhat-1",
           "comment": ""
         },
         {
           "component_name": "httpcomponents",
```

Comment:
There are already three different components related to this one: 

- apache-httpasyncclient
- apache-httpclient 
- apache-httpcore

 Each of them with their own versions. I would suggest leaving httpcomponents out of the final list.